src/services/plugin.service.ts

```diff
@@ -21,6 +21,11 @@ export class PluginService {
 
   public async initPluginState(container: IContainer): Promise<void> {
 
+    if (!process.env.MONGO_DB_NAME || !process.env.MONGO_URL ||
```

Comment:
prefer using `Environment`, not `process.env`

Reply:
`Environment` was removed

Reply:
seethe